cpp/benchmarks/perftest.cpp

```diff
@@ -185,6 +190,7 @@ ucs_status_t parseCommand(app_context_t* app_context, int argc, char* const argv
           return UCS_ERR_INVALID_PARAM;
         }
         break;
+      case 'e': app_context->endpoint_error_handling = false; break;
```

Comment:
I think you want `-e` to turn on endpoint error handling (since the default is no error handling?), so:

```suggestion
      case 'e': app_context->endpoint_error_handling = true; break;
```